net.sourceforge.qtpfsgui.LuminanceHDR.yaml

```diff
@@ -0,0 +1,99 @@
+app-id: net.sourceforge.qtpfsgui.LuminanceHDR
+runtime: org.kde.Platform
+runtime-version: "5.11"
+sdk: org.kde.Sdk
+command: luminance-hdr
+rename-desktop-file: luminance-hdr.desktop
+rename-appdata-file: luminance-hdr.appdata.xml
+rename-icon: luminance-hdr
+finish-args:
+  - --socket=fallback-x11
```

Comment:
you need --share=ipc if you want to use X11: http://docs.flatpak.org/en/latest/sandbox-permissions-reference.html?highlight=ipc#f1

Reply:
Thank you. Was not careful enough when reading that part of documentation